src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -2,25 +2,82 @@
 
 import org.springframework.stereotype.Service;
 
+/**
+ * Service class providing basic calculator operations.
+ */
 @Service
 public class CalculatorService {
 
```

Comment:
9. **Handling Decimal Inputs**:
   - If the calculator service is expected to handle decimal inputs, consider changing the method signatures to accept `double` or `BigDecimal` instead of `int`.